test/decorators.test.ts

```diff
@@ -96,6 +96,13 @@ describe('decorators', () => {
       .expect(code200, done);
   });
 
+  it('decorator @header', (done) => {
+    request(server)
+      .get('/header')
+      .expect(code200)
+      .expect('bar', done);
```

Comment:
you should  check the header instead of body.

Reply:
I returned the contents of the header, this is the same

Reply:
sorry，i saw wrongly